src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double fstSide;
+    private double secondSide;
+
+    public void setASide(double firstSide) {
+        this.fstSide = firstSide;
+    }
+
+    public void setBSide(double secondSide) {
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public void draw() {
+        setFigureName("Rectangle");
+        System.out.println("Figure: " + getFigureName());
+        setFigureArea(fstSide * secondSide);
```

Comment:
Logic of area calculation is common for all classes I suggest to make an interface, something loke AreaCalculator and define calculateArea method there